### PR TITLE
Feature/us3185 trip participants message

### PR DESCRIPTION
**Pull request by jpc11623**

### CODE REVIEW

File: Gateway/MinistryPlatform.Translation/Services/DonationService.cs [hunk 1]
```diff
@@ -481,7 +499,57 @@ public void SendMessageFromDonor(int pledgeId, string message)
                 ToContacts = toContacts,
                 MergeData = new Dictionary<string, object>()
             };
-            _communicationService.SendMessage(comm);
+            var communicationId = _communicationService.SendMessage(comm, true);
+            AddCommunicationDonation(donationId, communicationId);
```
Comment (IngageBrewerM) [inline Gateway/MinistryPlatform.Translation/Services/DonationService.cs]: Change to consistent name

File: CI/SQL/20160219144328_US3185-DonationCommunicationsPage.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+           ,'Donation Communication'
+           ,'Communication Messages for Donations'
+           ,100
+           ,'cr_Donation_Communications'
+           ,'Donation_Communications_ID'
+           ,1
+           ,'Donation_Communications_ID, Donation_ID, Communication_ID'
+           ,'cr_Donation_Communications.Donation_Communications_ID'
+           ,0)
+END
```
Comment (IngageBrewerM) [inline CI/SQL/20160219144328_US3185-DonationCommunicationsPage.sql]: Add insert statement to put this page in the Stewardship page section, which is page_id=9.
`INSERT INTO [dbo].[dp_Page_Section_Pages]([Page_ID],[Page_Section_ID],[User_ID]) VALUES(...);`

File: CI/SQL/20160219144328_US3185-DonationCommunicationsPage.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
+USE [MinistryPlatform]
+GO
+
+IF NOT EXISTS(SELECT * FROM [dbo].[dp_Pages] WHERE [Display_Name] = 'Donation Communications')
+BEGIN
+INSERT INTO [dbo].[dp_Pages]
+           ([Display_Name]
```
Comment (IngageBrewerM) [inline CI/SQL/20160219144328_US3185-DonationCommunicationsPage.sql]: Need to set to specific page id based on the standard we want to follow.

File: Gateway/MinistryPlatform.Translation/Services/DonationService.cs [hunk 4]
```diff
@@ -453,13 +471,13 @@ public void SendMessageToDonor(int donorId, int donationDistributionId, int from
...
 
-        public void SendMessageFromDonor(int pledgeId, string message)
+        public void SendMessageFromDonor(int pledgeId, int donationId, string message)
         {
             var toDonor = _pledgeService.GetDonorForPledge(pledgeId);
             var donorContact = _donorService.GetEmailViaDonorId(toDonor);
             var template = _communicationService.GetTemplate(_tripDonationMessageTemplateId);
 
-            var toContacts = new List<Contact> {new Contact {ContactId = donorContact.ContactId, EmailAddress = donorContact.Email}};
+            var toContacts = new List<Contact> { new Contact { ContactId = donorContact.ContactId, EmailAddress = donorContact.Email } };
```
Comment (IngageMaddoxT) [inline Gateway/MinistryPlatform.Translation/Services/DonationService.cs]: excess white space.  use resharper to format.  ctrl e + ctrl c

File: Gateway/crds-angular/MinistryPlatform.config [hunk 5]
```diff
@@ -10,7 +10,11 @@
   <add key="Attributes" value="277" />
   <add key="AttributesPageView" value="2185" />
   <add key="Batches" value="282" />
-  <add key="ChangePassword" value="475" /> 
+  <add key="ChangePassword" value="475" />
```
Comment (IngageMaddoxT) [inline Gateway/crds-angular/MinistryPlatform.config]: should we add these values to the other ministryplatform.config?

**Pull request merged**